src/app/shared/palette/order-summary/read-order-summary-field.html

```diff
@@ -0,0 +1,14 @@
+<div class="order-summary-title">Order Summary</div>
+<table>
+    <thead>
+      <tr><td>Code</td><td>Description</td><td>Amount</td></tr>
+    </thead>
+    <tbody>
+        <tr ccdReadOrderSummaryRow *ngFor="let feeValue of getFees()" [feeValue]="feeValue"></tr>
+        <tr>
+            <td></td>
+            <td class="payment-total">Total</td>
+            <td><ng-container #paymentTotal></ng-container></td>
```

Comment:
instead of manually inserting the ccd-read-money-gbp-field template in here from inside the component, can't we just do:
<td><ccd-read-money-gbp-field ....>

Reply:
No we cannot as we can never render the styles properly and no browser would render table correctly if we have custom tags inside table tag.